### ISSUE TITLE
Docs menu bug on mobile

### ISSUE THREAD

**Issue by apetta** [nandorojo/zeego#71]
![Screenshot_20231129_080122_Chrome.jpg](https://github.com/nandorojo/zeego/assets/33671231/0792e809-d402-45fe-a504-a793b61a6851)

The drawer menu is hidden under the main content on mobile

**Comment by apetta** [comment]
Actually, it seems to be something a few of your sites on mobile seem to be suffering from.

Solito:
![Screenshot_20231129_095338_Chrome.jpg](https://github.com/nandorojo/zeego/assets/33671231/afa8b649-0a97-4c75-876a-6d82e38ed1b6)



Moti:
![Screenshot_20231129_095459_Chrome.jpg](https://github.com/nandorojo/zeego/assets/33671231/fd56d42e-a21c-4710-a2d6-a151726d55dd)

**Comment by nandorojo** [comment]
Yeah it’s some custom CSS issue. Would be open to a PR fixing it.
